forge/ee/emailTemplates/LicenseExpired.js

```diff
@@ -0,0 +1,37 @@
+// Inserts:
+// - user.name
+// - forgeURL
+
+module.exports = {
+    subject: 'Your FlowForge license has expired',
+    text:
+`Hello {{{user.name}}},
+
+Your FlowForge License has now expired.
+
+You can apply a new license by logging into your account at
+{{#if forgeURL}}{{{forgeURL}}}{{else}}https://flowforge.com{{/if}}.
```

Comment:
`forgeURL` must be set. The only case where it won't be set is when running localfs driver - in which case they have no account to login into at flowforge.com so the message doesn't make sense.

Reply:
resolved in [08e27b4](https://github.com/flowforge/flowforge/pull/1757/commits/08e27b435eb25847cdd2ccbcdfbe9c9122f7a707)